test/test_collection.py

```diff
@@ -2265,6 +2265,10 @@ def test_find_command_generation(self):
                  ('$dumb', 2),
                  ('filter', {'foo': 1})]).to_dict())
 
+    def test_bool(self):
+        with self.assertRaises(NotImplementedError):
+            Collection(self.db, 'test').__bool__()
```

Comment:
I think it would be more "Pythonic" to call bool() instead of `.__bool__()`

Reply:
Done.